### PR TITLE
Fix typo s/destroied/destroyed/

### PR DESCRIPTION
**Pull request by mapreri**

### CODE REVIEW

File: src/curl.cpp [hunk 1]
```diff
@@ -1622,7 +1622,7 @@ bool S3fsCurl::CreateCurlHandle(bool force)
       S3FS_PRN_ERR("could not destroy handle.");
       return false;
     }
-    S3FS_PRN_INFO3("already has handle, so destroied it.");
+    S3FS_PRN_INFO3("already has handle, so destroyed it.");
```
Comment (andrewgaul) [inline src/curl.cpp]: Should this be "destroy"?

Comment (mapreri) [inline src/curl.cpp]: Dunno, I just followed the original tense.

Comment (driskell) [inline src/curl.cpp]: Since the destruction occurred before the message, I would say keep the existing tense, `destroyed`.

Comment (ggtakec) [inline src/curl.cpp]: Thanks all, I'm sorry for my english.

**Pull request merged**